### ISSUE TITLE
-Wsometimes-uninitialized in drivers/gpu/drm/amd/amdgpu/gfx_v9_0.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#936]
```
drivers/gpu/drm/amd/amdgpu/gfx_v9_0.c:754:6: warning: variable 'shadow' is used uninitialized whenever 'if' condition is
false [-Wsometimes-uninitialized]
        if (offset == grbm_cntl || offset == grbm_idx)
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
drivers/gpu/drm/amd/amdgpu/gfx_v9_0.c:757:6: note: uninitialized use occurs here
        if (shadow) {
            ^~~~~~
drivers/gpu/drm/amd/amdgpu/gfx_v9_0.c:754:2: note: remove the 'if' if its condition is always true
        if (offset == grbm_cntl || offset == grbm_idx)
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
drivers/gpu/drm/amd/amdgpu/gfx_v9_0.c:738:13: note: initialize the variable 'shadow' to silence this warning
        bool shadow;
                   ^
                    = 0
1 warning generated.
```

Patch submitted: https://lore.kernel.org/lkml/20200318002500.52471-1-natechancellor@gmail.com/

**Comment by nickdesaulniers** [comment]
accepted: https://cgit.freedesktop.org/~agd5f/linux/commit/?h=amd-staging-drm-next&id=b62984f50e4801ae105f5b47c8b6028dc08d59c5

**Comment by nathanchance** [comment]
Fixed in next-20200320: https://git.kernel.org/next/linux-next/c/931971280cf807b3def47ecf09af8419c2b953df
